daffodil-lib/src/main/resources/org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd

```diff
@@ -103,7 +103,8 @@
       </dfdl:defineFormat>
       
       <dfdl:defineFormat name="GeneralFormat">
-        <dfdl:format ref="GeneralFormatOriginal" />
+        <dfdl:format ref="GeneralFormatOriginal"
+            emptyElementParsePolicy="treatAsEmpty"/>
```

Comment:
I see you moved the property from GeneralFormatOriginal to GeneralFormat in this update. I guess this is so that the property is not defined in GeneralFormatPortable, so as long as schemas use the portabale format, IBM DFDL works as expected?

Reply:
Yes.